emmanuelmacharia/Store-Manager-databaseIntegration
# [#161529815] Fixed errors in the user.py files

### What does this Pull Request do?
Creates endpoints for the user such as login, register and sign out
### Description of task to be completed:
`POST api/v2/auth/signup` -Endpoint for registration
`GET api/v2/auth/signup` - Endpoint for getting all users
`DELETE api/v2/auth/signup` - Endpoint for deleting users
`POST api/v2/auth/login` - Endpoint for logging in
`DELETE api/v2/auth/logout` - Endpoint for logging out

### How it should be manually tested
* `git clone` this repository
* `cd/Store-Manager`
* Install a virtual environment
* on your preferred terminal, type `pip freeze > requirements.txt`
* Use `python run.py` or `flask run` to run the application 
* Start Postman and type in the URLs  registered in the `app/__init__.py` file, ensure you start with the `api/v2` prefix

### Relevant Pivotal Tracker stories
[#161495303](https://www.pivotaltracker.com/n/projects/2203951)
